### PR TITLE
Assume all database db tars are not enclosed in a folder.

### PR DESCRIPTION
**Pull request by yesimon**
The tar should directly contain the db files and subdirectories. This is to worry less about what
the containing folder is named.

### CODE REVIEW

**Comment by yesimon** [comment]
I can retar the existing tarball dbs. Can't change old code though obviously.

**Comment by dpark01** [comment]
Don't mutate any existing assets on GCS or DNAnexus that folks have been using -- just add new versions of the same side by side. I think this breaking change will require enough coordination across the installed user base that I'm hesitant to merge it quickly, let me think a bit.

Additionally, is this change compatible with the way metagenomics.krakenuniq_build currently produces databases and the way that our `dsub` based documentation (that calls metagenomics.py followed by `tar -C`) produces tarballs?

File: pipes/WDL/tasks/tasks_metagenomics.wdl [hunk 1]
```diff
@@ -16,16 +16,17 @@ task krakenuniq {
...
-    DB_DIR=$(mktemp -d)
+    DB_DIR=$(mktemp -d --suffix _db)
+    mkdir -p $DB_DIR/krakenuniq $DB_DIR/krona
 
     metagenomics.py --version | tee VERSION
 
     # decompress DB to $DB_DIR
     read_utils.py extract_tarball \
-      ${krakenuniq_db_tar_lz4} $DB_DIR \
+      ${krakenuniq_db_tar_lz4} $DB_DIR/krakenuniq \
```
[the remark this answers is not in the document]
Comment (yesimon) [reply pipes/WDL/tasks/tasks_metagenomics.wdl:26]: Yep

**Comment by dpark01** [comment]
Okay that seems like it ought to work ... but for some reason it doesn't appear to be working. Here's a [CI job failure](https://platform.dnanexus.com/projects/F8PQ6380xf5bK0Qk0YPjB17P/monitor/job/FpGJ4v80xf5zqfvp10yP3qKj).

```
2020-04-16 22:08:22,223 - cmd:199:main_argparse - INFO - command: /opt/viral-ngs/source/read_utils.py extract_tarball tarfile=/home/dnanexus/inputs/krakenuniq.full.20180811.tar.zst out_dir=/tmp/tmp.bT8eJxOsTE_db/krakenuniq compression=auto pipe_hint=None threads=None loglevel=DEBUG tmp_dir=/tmp tmp_dirKeep=False
2020-04-16 22:08:22,224 - cmd:213:main_argparse - DEBUG - using tempDir: /tmp/tmp-read_utils-extract_tarball-cjoosd_z
2020-04-16 22:08:22,228 - file:271:extract_tarball - DEBUG - cat /home/dnanexus/inputs/krakenuniq.full.20180811.tar.zst | zstd -dc | tar -C /tmp/tmp.bT8eJxOsTE_db/krakenuniq -x --no-same-owner
CPU: 5% (32 cores) * Memory: 6534/255147MB * Storage: 970GB free * Net: 178↓/2↑MBps
2020-04-16 22:20:19,284 - file:287:extract_tarball - DEBUG - completed unpacking of /home/dnanexus/inputs/krakenuniq.full.20180811.tar.zst into /tmp/tmp.bT8eJxOsTE_db/krakenuniq
/tmp/tmp.bT8eJxOsTE_db/krakenuniq
+ '[' -d /tmp/tmp.bT8eJxOsTE_db/krakenuniq/krakenuniq ']'
+ mv '/tmp/tmp.bT8eJxOsTE_db/krakenuniq/krakenuniq/*' /tmp/tmp.bT8eJxOsTE_db/krakenuniq
mv: cannot stat '/tmp/tmp.bT8eJxOsTE_db/krakenuniq/krakenuniq/*': No such file or directory
```

**Comment by yesimon** [comment]
Can't seem to merge as I'm not an authorized user.

**Comment by dpark01** [comment]
Thanks -- I'll merge this in once this [CI test](https://platform.dnanexus.com/projects/F8PQ6380xf5bK0Qk0YPjB17P/monitor/analysis/FpGpz8Q0xf5q12kG2Q6Yf8kB) clears.

**Pull request merged**